tiled/client/utils.py

```diff
@@ -14,11 +14,54 @@
 MSGPACK_MIME_TYPE = "application/x-msgpack"
 
 
+def raise_for_status(response) -> None:
+    """
+    Raise the `httpx.HTTPStatusError` if one occurred. Include correlation ID.
+    """
+    # This adapted from the method httpx.Response.raise_for_status, modified to
+    # remove the generic link to HTTP status documentation and include the
+    # correlation ID.
+    request = response._request
+    if request is None:
+        raise RuntimeError(
+            "Cannot call `raise_for_status` as the request "
+            "instance has not been set on this response."
+        )
+
+    if response.is_success:
+        return
```

Comment:
Should `response` be returned to be consistent with `httpx.Response.raise_for_status`?

https://github.com/encode/httpx/blob/392dbe45f086d0877bd288c5d68abf860653b680/httpx/_models.py#L739-L740

Reply:
Ah, yeah, this recently [changed](https://github.com/encode/httpx/blob/master/CHANGELOG.md#0250-11th-september-2023), and I was going off of a slightly old version of the source. Good catch.

Fix pushed.